en/Developers/Build plugins.md

```diff
@@ -0,0 +1,60 @@
+Learn how you can extend Obsidian with more features by building your own plugins and sharing them with the community on the [[Marketplace]].
+
+## Quickstart
```

Comment:
Is "quickstart" a valid one-word term?

Reply:
Grammatically, "quick start" is more correct, but "quickstart" has become synonymous with this kind of document and is usually spelled without space. For example, [GitHub](https://docs.github.com/en/get-started/quickstart).